blank_project/assembly/main.ts

```diff
@@ -1,6 +1,4 @@
-import { context, storage, near } from "./near";
-
-import { Greeter } from "./model.near";
+import { Greeter } from "../out/model.near";
```

Comment:
@willemneal will this work fine when `./model` has some dependencies within project?

Reply:
With the current setup you have to create the near file from model, so I'm not exactly sure.  I think to make that work, we either have to output the near files into the same directory as the source or copy over all of the imports.

I really want to remove the step of having to create temporary files from the tool chain.  I posed an idea to Bowen of using decorators on classes to tell the compiler that the class should be transformed during compilation.

Reply:
Looks like you already use `process.cwd()` in the same file:
```
readFile: (filename, baseDir) => {
        baseDir = pathModule.relative(process.cwd(), baseDir);
```